src/app/technical-record/ministry-plates/ministry-plates.component.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  Component,
+  OnInit,
+  ChangeDetectionStrategy,
+  Input,
+  Output,
+  EventEmitter
+} from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+
+import { PLATE_ISSUE_VALUES } from '@app/app.enums';
+import { UserService } from '@app/app-user.service';
+import { Applicant, Plate } from '@app/models/tech-record.model';
+import { UserDetails } from '@app/models/user-details';
+
+@Component({
+  selector: 'vtm-ministry-plates-template',
+  templateUrl: './ministry-plates.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  exportAs: 'plateComponent'
+})
+export class MinistryPlatesComponent implements OnInit {
+  @Input() applicantDetails: Applicant;
+  @Output() plateHandler = new EventEmitter<Plate>();
+  platesForm: FormGroup;
+  issueValues = Object.values(PLATE_ISSUE_VALUES);
+
+  constructor(public loggedUser: UserService) {}
+
+  ngOnInit() {
+    const plateIssuer = !!this.loggedUser.getUser()
+      ? this.loggedUser.getUser().msUser
+      : ({} as UserDetails);
```

Comment:
`this.loggedUser.getUser().msUser` should be sufficient. The value should be sent for you to take this plate action.